packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -210,14 +210,41 @@ async function getNpmTarballUrl(packageName: string, version: string = 'latest')
   return stdout.trim();
 }
 
+/**
+ * Gets expo SDK version major from the local package.json.
+ */
+async function getLocalSdkMajorVersion(): Promise<string | null> {
+  const path = require.resolve('expo/package.json', { paths: [process.cwd()] });
+  if (!path) return null;
```

Comment:
Please, stop doing that 😛 
```suggestion
  if (!path) {
    return null;
  }
```

Reply:
sorry 🥺